### PR TITLE
SLIM-961 Make AddDevice Applicable For GasMeters

### PR DESCRIPTION
**Pull request by riannefijn**

### CODE REVIEW

**Comment by riannefijn** [comment]
retest this please

**Comment by riannefijn** [comment]
retest this please

**Comment by riannefijn** [comment]
retest this please

**Comment by riannefijn** [comment]
retest this please

**Comment by robindenadel** [comment]
retest this please

File: cucumber-tests-platform/src/test/java/com/alliander/osgp/cucumber/platform/PlatformKeys.java [hunk 1]
```diff
@@ -309,4 +309,9 @@
 
     public static final String DATE = "Date";
     public static final String TIME_UNTIL_ON = "TimeUntilOn";
+
+    public static final String MBUS_IDENTIFICATION_NUMBER = "MbusIdentificationNumber";
+    public static final String MBUS_MANUFACTURER_IDENTIFICATION = "MbusManufacturerIdentification";
+    public static final String MBUS_VERSION = "MbusVersion";
+    public static final String MBUS_DEVICE_TYPE_IDENTIFICATION = "MbusDeviceTypeIdentification";
```
Comment (robindenadel) [inline cucumber-tests-platform/src/test/java/com/alliander/osgp/cucumber/platform/PlatformKeys.java]: PlatformSmartmeteringKeys is a better class for these constants

**Pull request merged**